tests/filecheck/dialects/irdl/testd.irdl.mlir

```diff
@@ -144,5 +144,39 @@ builtin.module {
       irdl.operands(%0, single %0, optional %0, variadic %0)
       irdl.results(%0, single %0, optional %0, variadic %0)
     }
+
+    // CHECK:      irdl.operation @op_with_regions {
+    // CHECK-NEXT:   %{{.*}} = irdl.region
+    // CHECK-NEXT:   %{{.*}} = irdl.region()
+    // CHECK-NEXT:   %{{.*}} = irdl.is i32
+    // CHECK-NEXT:   %{{.*}} = irdl.is i64
+    // CHECK-NEXT:   %{{.*}} = irdl.region(%{{.*}}, %{{.*}})
+    // CHECK-NEXT:   %{{.*}} = irdl.region with size 3
+    // CHECK-NEXT:   irdl.regions(%{{.*}}, %{{.*}}, %{{.*}}, %{{.*}})
+    // CHECK-NEXT: }
+    irdl.operation @op_with_regions {
+      %r0 = irdl.region
+      %r1 = irdl.region()
+      %v0 = irdl.is i32
+      %v1 = irdl.is i64
+      %r2 = irdl.region(%v0, %v1)
+      %r3 = irdl.region with size 3
+
+      irdl.regions(%r0, %r1, %r2, %r3)
+    }
+
+    // CHECK:      irdl.operation @attr_op {
+    // CHECK-NEXT:   %{{.*}} = irdl.any
+    // CHECK-NEXT:   %{{.*}} = irdl.is i64
+    // CHECK-NEXT:   irdl.attributes { "attr1" = %{{.*}}, "attr2" = %{{.*}} }
+    // CHECK-NEXT: }
+    irdl.operation @attr_op {
+      %0 = irdl.any
+      %1 = irdl.is i64
+      irdl.attributes {
+        "attr1" = %0,
+        "attr2" = %1
```

Comment:
how does this work?

Reply:
Oh I see it's fully custom syntax

Reply:
yes, I also wasn't sure if it should be printed in this way or if the single line print is acceptable?

Reply:
I think this looks nicer

Reply:
Ok I'll try to figure out how new lines/indentation work